README.md

```diff
@@ -0,0 +1,61 @@
+# Install Conan Action
+
+[![Validate Conan Action](https://github.com/conan-io/setup-conan/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/conan-io/setup-conan/actions/workflows/ci.yml)
+[![GitHub Marketplace](https://img.shields.io/badge/Marketplace-Setup%20Conan%20Client-blue.svg?colorA=24292e&colorB=0366d6&style=flat&longCache=true&logo=github)](https://github.com/marketplace/actions/setup-conan-client)
+
+
+A GitHub Action to install and configure the Conan package manager in your workflow.
+This action provides a simple and flexible way to set up Conan with custom configurations and audit settings.
+
+## Features
+
+- 🚀 Install any version of Conan 2.x
+- ⚙️ Apply custom Conan configurations
+- 🔐 Configure audit tokens
```

Comment:
```suggestion
- 🔐 Configure audit tokens for dependencies vulnerabilities scanning
```
Or something like that, as ``audit`` might not be well known yet?

Reply:
I commented with Carlos, as future steps, adding a banner for Conan audit in the README and a gif with conan audit running with this GH action